.github/workflows/contracts-deploy-sepolia.yml

```diff
@@ -7,7 +7,8 @@ jobs:
   deploy-contract:
     runs-on: ubuntu-latest
     env:
-      CONTRACT_PATH: src/FactChainCommunity.sol:FactChainCommunity
+      FC_COMMUNITY_CONTRACT_PATH: src/FactChainCommunity.sol
```

Comment:
Might be even simpler to copy the whole file, and have `deploy-fc-community-contract` and `deploy-fc-nft-contract`.
This way deployment is in parallel, if one fails, the other can still deploy, the `Log detail of deployment` and `upload of artifact` will be working etc.

I would also replace `CONTRACT_PATH` by `CONTRACT_NAME` and derive the path to the `.sol` and json abi from that.